Fb footer

silveranstavern opened the pull request.

silveranstavern committed:
Corrected footer pathing

Corrected footer pathing to reflect its new location after being dragged and dropped into the components folder.

silveranstavern committed:
Modify app.component.ts

New code adds footer section, and a comment where header and routs will be rendered as well.

silveranstavern committed:
Merge branch 'master' into fb-footer

# Conflicts:
#	StackedAngular/src/app/app.module.ts

silveranstavern committed:
Merge branch 'master' into fb-footer

# Conflicts:
#	StackedAngular/src/app/app.module.ts

JGrippo reviewed (changes requested):
There are a bunch of backup files inside of the PR.
Also in your component template you reference app-header which I don't think exists yet.

silveranstavern committed:
Delete git tooling autogen files

Updated gitignore and removed autogenerated files that resovled merge conflicts.

JGrippo commented:
Changes required.
